system/include/webgpu/webgpu_cpp.h

```diff
@@ -0,0 +1,1155 @@
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+#ifndef WEBGPU_CPP_H_
```

Comment:
I don't think we should put webgpu_cpp.h in emscripten: the true webgpu interface is in webgpu.h and webgpu_cpp can be compiled in applications themselves.